firmware/config/boards/microrusefi/!compile-mre.bat

```diff
@@ -3,16 +3,8 @@ cd ../../..
 
 set LDSCRIPT = config/boards/NUCLEO_F767/STM32F76xxI.ld
 
-set PROJECT_BOARD=nucleo_f767
+set PROJECT_BOARD=microrusefi
 set PROJECT_CPU=ARCH_STM32F7
-set EXTRA_PARAMS=-DDUMMY ^
- -DSTM32F767xx ^
- -DEFI_USE_OSC=TRUE ^
- -DFIRMWARE_ID=\"microRusEfi\" ^
- -DEFI_COMMUNICATION_PIN=GPIOE_1 ^
- -DEFI_FATAL_ERROR_PIN=GPIOE_2 ^
- -DEFI_RUNNING_PIN=GPIOE_3 ^
- -DEFI_WARNING_PIN=GPIOE_4
```

Comment:
in general we should set this stuff in the `.mk` when we can - it's far sturdier and more portable